live-examples/js-examples/globalprops-eval.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">function equivalence(x,y){
+  if (eval(x) === eval(y)) return "Strong";
+  if (eval(x.toString()) === eval(y.toString())) return "Weak";
```

Comment:
In terms of code style, I am wondering if we should follow the project wide style for `if` conditionals i.e. The above should be written as:

```
if (eval(x) === eval(y)) {
    return "Strong";
}

if (eval(x.toString()) === eval(y.toString())) {
    return "Weak";
}
```

Reply:
I agree that it's probably better form to write them the way you're doing, the issue is space. Actually now I think of it in more recent examples (ie numbers) I've been doing stuff like:

```
if (isNaN(x)) { return 0 }
```
So we should stick a style and I should stick with it :)

Reply:
I completely agree. I also entirely get the number of lines problem so, it is a balance we need to find. I am leaning towards to first example of the second but, would like to know @wbamberg thoughts on this.

Reply:
I think we should have:

```
if (eval(x) === eval(y)) {
    return "Strong";
}
```

I feel pretty strongly that we should not sacrifice readability for compactness.

What would be lovely would be to have:
(1) a style guide for JS code samples, not just for this but for all samples on MDN
(2) lint rules to enforce it

The nearest thing I have found is https://developer.mozilla.org/en-US/docs/MDN/Contribute/Guidelines/Code_samples#JavaScript but it's pretty limited.

Reply:
@schalkneethling , does webdev have guidelines we can steal?

Reply:
There is this, but I am pretty sure they are outdated :( https://mozweb.readthedocs.io/en/latest/reference/js-style.html